### PR TITLE
Branding: Add Microsoft logo to first loading screen

### PR DESCRIPTION
**Pull request by kimprice**
The following loading screen is displayed on first load for all targets instead of a target-specific icon.
microsoftLoad

### CODE REVIEW

**Comment by Jaqster** [comment]
Will we still keep the theme colors?  i.e. blue for micro:bit and dark grey for minecraft? 
image
image

**Comment by kimprice** [comment]
@Jaqster great question
Yes, it will keep the theme color. I just checked micro:bit:
image

File: theme/themes/pxt/elements/loader.overrides [hunk 1]
```diff
@@ -15,11 +15,16 @@
...
   background: transparent @loaderImage no-repeat center center;
-  background-size: 100%;
+  background-size: @loaderImageSize;
  -webkit-animation: @loaderAnimation @loaderSpeed infinite linear;
  animation: @loaderAnimation @loaderSpeed infinite linear;
 }
 
+.ui.loader.main.msft:after {
+  background: transparent @loaderMsftImage no-repeat center center;
+  background-size: 99%;
```
Comment (jwunderl) [inline theme/themes/pxt/elements/loader.overrides:25]: does the 1% here make a difference / what's the change here handling?

File: theme/themes/pxt/elements/loader.overrides [hunk 2]
```diff
@@ -15,11 +15,16 @@
...
   border-radius: 0px;
   background: transparent @loaderImage no-repeat center center;
-  background-size: 100%;
+  background-size: @loaderImageSize;
  -webkit-animation: @loaderAnimation @loaderSpeed infinite linear;
  animation: @loaderAnimation @loaderSpeed infinite linear;
 }
 
+.ui.loader.main.msft:after {
+  background: transparent @loaderMsftImage no-repeat center center;
```
Comment (jwunderl) [inline theme/themes/pxt/elements/loader.overrides:24]:
```suggestion
  background-image: @loaderMsftImage;
```

should work fine with a little less duped? doesn't really matter though, either way works

**Review by jwunderl — APPROVED** [review]
1 question / one possible suggestion but otherwise lgtm

File: theme/themes/pxt/elements/loader.overrides (hunk 1 above)
Comment (jwunderl) [reply theme/themes/pxt/elements/loader.overrides:25]: Ah okay makes sense, might be worth a quick comment above it so if we run into it again we know why it's special cased a bit~

**Review by jwunderl — APPROVED** [review]

**Comment by THEb0nny** [comment]
@kimprice you need to make @loaderAnimation separate, as I understand it, if you override @loaderAnimation, then the animation for the Microsoft logo will be the same as for the project logo.

**Comment by kimprice** [comment]
> @kimprice you need to make @loaderAnimation separate, as I understand it, if you override @loaderAnimation, then the animation for the Microsoft logo will be the same as for the project logo.

Yes, this change is just to change the initial loading screen. All other loading screens will be target-specific. I override the `@loaderImageUrl` in Arcade to change the target-specific loading screen (https://github.com/microsoft/pxt-arcade/pull/5987/files#diff-a71b80d431a659366ab01f6d02dc9b1fc7a5f3a6cfe4cccf65df4450839aafc8).

**Comment by THEb0nny** [comment]
@kimprice https://github.com/microsoft/pxt-ev3/pull/1030/commits/bf193b4e1ee319ebbd15869c668071c083c58988

**Pull request merged**